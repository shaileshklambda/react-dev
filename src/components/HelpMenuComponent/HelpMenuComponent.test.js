import React from "react";
import renderer from "react-test-renderer";
import Helpmenucomponent from ".";

it("Helpmenucomponent: default", () => {
  const component = renderer.create(<Helpmenucomponent />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
