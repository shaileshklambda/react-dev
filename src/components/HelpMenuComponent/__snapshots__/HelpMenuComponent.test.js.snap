// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Helpmenucomponent: default 1`] = `
<div>
  <img
    className=""
    height={20}
    onClick={[Function]}
    src={
      Object {
        "ReactComponent": [Function],
        "__esModule": true,
        "default": "help-center.svg",
      }
    }
    width={25}
  />
</div>
`;
